src/reducers/feeds.js

```diff
@@ -74,6 +74,7 @@ function convert(valueType, value) {
     case PRICE_WITH_SAFETY_MARGIN:
       return fromRay(value, 5);
     case DEBT_CEILING:
+      return fromRad(value, 0);
```

Comment:
@levity I'm confused if this should be `fromWei` or `fromRad`, I set it to `rad` based on a comment that the line value that we get back is expressed `$LINE * 10^45`. Can you confirm?

Reply:
If the values you see when you use `fromRad` look correct, then stick with it :) I just copied things over from the previous code, but it could have always been wrong (or have been made wrong by changes in dss) but we never noticed since it wasn't displayed. I see that the global debt ceiling is also in rad